tests/js/sentry-test/reactTestingLibrary.tsx

```diff
@@ -112,7 +113,12 @@ function render(ui: React.ReactElement, options?: Options) {
 const fireEvent = rtl.fireEvent;
 
 function renderGlobalModal(options?: Options) {
-  const result = render(<GlobalModal />, options);
+  const result = render(
+    <div id={ROOT_ELEMENT}>
```

Comment:
This is necessary to add the escape key handler in global modal https://github.com/getsentry/sentry/blob/8d8eca63e1f055830f1c22d1704163659470cc44/static/app/components/globalModal/index.tsx#L128

Reply:
On second thought, I'm going to revert this. It will be better to make sure the modal always adds the event listener so it doesn't matter if you use `renderGlobalModal` or not